### PR TITLE
updated script using the new getActualNick()

### PR DESCRIPTION
**Pull request by GandalfGuenter**
Sinusbot 0.9.12.2-58b509d or higher needed

### CODE REVIEW

**Comment by GandalfGuenter** [comment]
Note: I didn't test this, but it should work

File: scripts/nickCrashHelper.js [hunk 1]
```diff
@@ -65,25 +65,10 @@ registerPlugin({
...
-                j++;
-            }
-
-            nick = nick.substring(0, length - j);
-
-            if (j != 0) {
-                debug("Nick set to " + nick + " (found " + j + " crash-numbers after the nick.)");
-            } else {
-                debug("Nick not modified, all seems normal :)");
+            if (sinusbot.getNick() == sinusbot.getActualNick()) { // 0.9.12.2-58b509d or higher needed
```
Comment (flyth) [inline scripts/nickCrashHelper.js]: Sure you don't mean !=?

Comment (xDefcon) [inline scripts/nickCrashHelper.js]: For sure the actual nickname must be different from the real one to set the right nick.

Will correct that small problem, do some tests, and commit new version in about 1hr.

Comment (GandalfGuenter) [inline scripts/nickCrashHelper.js]: yes, I made a typo ^^

**Comment by xDefcon** [comment]
@GandalfGuenter Also another enanchment, in your commit you called 3 times the function getNick() and 2 times getActualNick(), in this case is not needed, because that will return the same value of the previous call. So it is better to declare 2 variables :smiley:

**Comment by GandalfGuenter** [comment]
@xDefcon hmm, doesn't really matter. I personally think its just more unnecessary code...

**Pull request closed without merging**